### ISSUE TITLE
strange matplotlib force plots

### ISSUE THREAD

**Issue by cbeauhilton** [shap/shap#625]
I'm having a strange recurrent issue with the matplotlib force plots. See the image for an example - the text rotation seems to start at the wrong place, leading to the text overrunning the graph.

![SHAP_forceplot](https://user-images.githubusercontent.com/34774299/58711642-f294aa00-838c-11e9-814c-8248b4a9cb2d.png)

```
python 3.7.3
shap 0.28.5
matplotlib 3.1.0
```

Here's the code that made that example:

```python
 shap.force_plot(
                            expected_value,  
                            shap_values[pt_num, :],
                            features_shap.iloc[
                                pt_num, :
                            ],  
                            text_rotation=15,  
                            matplotlib=True,  # instead of Javascript
                            show=False,  
                            link="logit",
                            feature_names=features_shap.columns,
                        )
                        plt.savefig(
                            (title),
                            dpi=1200,
                            transparent=True,
                            bbox_inches="tight",
                            format='png',
                        )
```
I thought maybe my mpl rcParams had gotten screwed up, so I made a fresh Conda environment with basically only shap and its dependencies. I also tried making the images inline in a Jupyter notebook rather than from vanilla .py files, and outputting to various filetypes (png, pdf, ps, etc.), tried all of the mpl backends, but it's the same. The JS plots are fine. 

Is anyone else having this issue? It cropped up rather unexpectedly a few days ago, and I'm not sure what I could have changed that would have caused this.

Thanks!

**Comment by koomie** [comment]
Same issue happens for me in latest `0.33.0` release.

**Comment by slundberg** [comment]
Fixed by #947. Thanks @koomie !
